normandy/control/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+from django.conf.urls import url, include
+
+from normandy.control import views as control_views
+from django.contrib.auth.views import login, logout_then_login
+
+urlpatterns = []
+
+if settings.ADMIN_ENABLED:
+  urlpatterns += [
+      url(r'^control/', include([
+          url(r'^$', control_views.IndexView.as_view(), name='index'),
```

Comment:
I'd imagine something like a namespace of `normandy` and a route name of `control.index`. I haven't looked a lot into namespaces though.